src/pages/students/[student_id]/studentIEP.tsx

```diff
@@ -0,0 +1,22 @@
+type Student = {
+  //do we need this here?
+  student_id: string;
+  first_name: string;
+  last_name: string;
+  email: string;
+  assigned_case_manager_id: string | null;
+};
```

Comment:
there is a way to derive this from trpc types, but since this component only uses the first and last name I'd recommend adjusting the props to take `{ first_name: string, last_name: string }` or similar instead of `{ student: Student }`

Reply:
I figured since this component was meant to display the IEP when we make it, it'd make sense to pass in more than just the student's name because we want to access the IEP information later. but I can still change this for now.